mcp/tests/staktrak/src/index.ts

```diff
@@ -507,8 +605,87 @@ class UserBehaviorTracker {
     // this listener only needs to be setup once
     if (this.memory.alwaysListeners.length > 0) return;
 
+    // Define action removal handlers with error handling
+    const actionRemovalHandlers: Record<string, (data: any) => boolean> = {
+      'staktrak-remove-navigation': (data) => {
```

Comment:
why are these `staktrak-remove-` msgs needed inside the iframe? Isn't the parent responsible for the recording and playback stuff?